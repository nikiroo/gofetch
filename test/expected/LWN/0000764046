                     SECURITY UPDATES FOR MONDAY                   

  [Security] Sep 3, 2018 15:41 UTC (Mon) (ris)

  o News link: https://lwn.net/Articles/764046
  o Source link: 


    Dist.
    
    ID
    
    Release
    
    Package
    
    Date
    
    Debian
    
    [1]DLA-1492-1
    
    LTS
    
    dojo
    
    2018-09-03
    
    Debian
    
    [2]DLA-1487-1
    
    LTS
    
    libtirpc
    
    2018-08-31
    
    Debian
    
    [3]DLA-1488-1
    
    LTS
    
    mariadb-10.0
    
    2018-08-31
    
    Debian
    
    [4]DLA-1490-1
    
    LTS
    
    php5
    
    2018-09-01
    
    Debian
    
    [5]DSA-4283-1
    
    stable
    
    ruby-json-jwt
    
    2018-08-31
    
    Debian
    
    [6]DLA-1488-1
    
    LTS
    
    spice
    
    2018-08-31
    
    Debian
    
    [7]DLA-1486-1
    
    LTS
    
    spice
    
    2018-09-01
    
    Debian
    
    [8]DLA-1489-1
    
    LTS
    
    spice-gtk
    
    2018-09-01
    
    Debian
    
    [9]DLA-1491-1
    
    LTS
    
    tomcat8
    
    2018-09-02
    
    Debian
    
    [10]DSA-4282-1
    
    stable
    
    trafficserver
    
    2018-08-31
    
    Fedora
    
    [11]FEDORA-2018-33fef25ed1
    
    F28
    
    ghc-hakyll
    
    2018-08-31
    
    Fedora
    
    [12]FEDORA-2018-33fef25ed1
    
    F28
    
    ghc-hs-bibutils
    
    2018-08-31
    
    Fedora
    
    [13]FEDORA-2018-07083800ac
    
    F28
    
    ghostscript
    
    2018-09-02
    
    Fedora
    
    [14]FEDORA-2018-77e610115a
    
    F28
    
    mariadb
    
    2018-08-31
    
    Fedora
    
    [15]FEDORA-2018-33fef25ed1
    
    F28
    
    pandoc-citeproc
    
    2018-08-31
    
    Fedora
    
    [16]FEDORA-2018-f2b24ce26e
    
    F28
    
    phpMyAdmin
    
    2018-08-31
    
    Fedora
    
    [17]FEDORA-2018-915602df63
    
    F27
    
    xen
    
    2018-08-31
    
    Mageia
    
    [18]MGASA-2018-0366
    
    6
    
    java-1.8.0-openjdk
    
    2018-09-02
    
    Mageia
    
    [19]MGASA-2018-0361
    
    6
    
    libarchive
    
    2018-08-31
    
    Mageia
    
    [20]MGASA-2018-0367
    
    6
    
    libgd
    
    2018-09-02
    
    Mageia
    
    [21]MGASA-2018-0356
    
    6
    
    libraw
    
    2018-08-31
    
    Mageia
    
    [22]MGASA-2018-0364
    
    6
    
    libxcursor
    
    2018-08-31
    
    Mageia
    
    [23]MGASA-2018-0359
    
    5
    
    mariadb
    
    2018-08-31
    
    Mageia
    
    [24]MGASA-2018-0355
    
    5, 6
    
    mercurial
    
    2018-08-31
    
    Mageia
    
    [25]MGASA-2018-0363
    
    6
    
    openssh
    
    2018-08-31
    
    Mageia
    
    [26]MGASA-2018-0365
    
    6
    
    openssl
    
    2018-09-02
    
    Mageia
    
    [27]MGASA-2018-0358
    
    6
    
    poppler
    
    2018-08-31
    
    Mageia
    
    [28]MGASA-2018-0362
    
    6
    
    quazip
    
    2018-08-31
    
    Mageia
    
    [29]MGASA-2018-0357
    
    6
    
    squirrelmail
    
    2018-08-31
    
    Mageia
    
    [30]MGASA-2018-0360
    
    6
    
    virtualbox
    
    2018-08-31
    
    openSUSE
    
    [31]openSUSE-SU-2018:2590-1
    
    42.3
    
    cobbler
    
    2018-09-03
    
    openSUSE
    
    [32]openSUSE-SU-2018:2592-1
    
    15.0
    
    libressl
    
    2018-09-03
    
    openSUSE
    
    [33]openSUSE-SU-2018:2587-1
    
    42.3
    
    wireshark
    
    2018-09-02
    
    openSUSE
    
    [34]openSUSE-SU-2018:2591-1
    
    15.0 42.3
    
    zutils
    
    2018-09-03
    
    SUSE
    
    [35]SUSE-SU-2018:2576-1
    
    OS7
    
    OpenStack
    
    2018-08-31
    
    SUSE
    
    [36]SUSE-SU-2018:2578-1
    
    OS7
    
    couchdb
    
    2018-08-31
    
    SUSE
    
    [37]SUSE-SU-2018:2574-1
    
    SLE11
    
    java-1_7_0-ibm
    
    2018-08-31
    
    SUSE
    
    [38]SUSE-SU-2018:2583-1
    
    SLE11
    
    java-1_7_1-ibm
    
    2018-08-31
    
    SUSE
    
    [39]SUSE-SU-2018:2584-1
    
    SLE12
    
    spice
    
    2018-08-31
    
    
    
    [1] https://lwn.net/Articles/764007/
    
    [2] https://lwn.net/Articles/764008/
    
    [3] https://lwn.net/Articles/764009/
    
    [4] https://lwn.net/Articles/764010/
    
    [5] https://lwn.net/Articles/764011/
    
    [6] https://lwn.net/Articles/764013/
    
    [7] https://lwn.net/Articles/764012/
    
    [8] https://lwn.net/Articles/764014/
    
    [9] https://lwn.net/Articles/764015/
    
    [10] https://lwn.net/Articles/764016/
    
    [11] https://lwn.net/Articles/764017/
    
    [12] https://lwn.net/Articles/764018/
    
    [13] https://lwn.net/Articles/764019/
    
    [14] https://lwn.net/Articles/764020/
    
    [15] https://lwn.net/Articles/764021/
    
    [16] https://lwn.net/Articles/764022/
    
    [17] https://lwn.net/Articles/764023/
    
    [18] https://lwn.net/Articles/764024/
    
    [19] https://lwn.net/Articles/764025/
    
    [20] https://lwn.net/Articles/764026/
    
    [21] https://lwn.net/Articles/764027/
    
    [22] https://lwn.net/Articles/764028/
    
    [23] https://lwn.net/Articles/764029/
    
    [24] https://lwn.net/Articles/764030/
    
    [25] https://lwn.net/Articles/764031/
    
    [26] https://lwn.net/Articles/764032/
    
    [27] https://lwn.net/Articles/764033/
    
    [28] https://lwn.net/Articles/764034/
    
    [29] https://lwn.net/Articles/764035/
    
    [30] https://lwn.net/Articles/764036/
    
    [31] https://lwn.net/Articles/764037/
    
    [32] https://lwn.net/Articles/764038/
    
    [33] https://lwn.net/Articles/764039/
    
    [34] https://lwn.net/Articles/764040/
    
    [35] https://lwn.net/Articles/764044/
    
    [36] https://lwn.net/Articles/764041/
    
    [37] https://lwn.net/Articles/764042/
    
    [38] https://lwn.net/Articles/764043/
    
    [39] https://lwn.net/Articles/764045/



