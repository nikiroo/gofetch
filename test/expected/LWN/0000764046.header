0Security updates for Monday	null/LWN/0000764046		70
i  [Security] Sep 3, 2018 15:41 UTC (Mon) (ris)
i
i    Security  updates  have  been issued by Debian (dojo, libtirpc,
i    mariadb-10.0,  php5,  ruby-json-jwt, spice, spice-gtk, tomcat8,
i    and   trafficserver),   Fedora   (ghc-hakyll,  ghc-hs-bibutils,
i    ghostscript,  mariadb,  pandoc-citeproc,  phpMyAdmin, and xen),
i    Mageia    (java-1.8.0-openjdk,   libarchive,   libgd,   libraw,
i    libxcursor,  mariadb,  mercurial,  openssh,  openssl,  poppler,
i    quazip,   squirrelmail,  and  virtualbox),  openSUSE  (cobbler,
i    libressl,   wireshark,   and   zutils),   and   SUSE  (couchdb,
i    java-1_7_0-ibm, java-1_7_1-ibm, OpenStack, and spice).
i
