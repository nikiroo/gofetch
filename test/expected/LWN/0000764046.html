<!DOCTYPE html>
<html>
<head>
  <meta http-equiv='content-type' content='text/html; charset=utf-8'>
  <meta name='viewport' content='width=device-width, initial-scale=1.0'>
  <style type='text/css'>
    body { margin: 1em 15%; }
  </style>
</head>
<body>
<div class='story'>
	<h2>Security updates for Monday</h2>
	<div class='details'>([Security] Sep 3, 2018 15:41 UTC (Mon) (ris))</div>
	<br/>
    <ul>
    <li>Reference: <a href=''>0000764046</a></li>
        <li>News link: <a href='https://lwn.net/Articles/764046'>https://lwn.net/Articles/764046</a></li>
        <li>Source link: <a href=''></a></li>
    </ul>
	<br/>
	<div class='content' style='text-align: justify'>
		Dist.<br/><br/>ID<br/><br/>Release<br/><br/>Package<br/><br/>Date<br/><br/>Debian<br/><br/>[1]DLA-1492-1<br/><br/>LTS<br/><br/>dojo<br/><br/>2018-09-03<br/><br/>Debian<br/><br/>[2]DLA-1487-1<br/><br/>LTS<br/><br/>libtirpc<br/><br/>2018-08-31<br/><br/>Debian<br/><br/>[3]DLA-1488-1<br/><br/>LTS<br/><br/>mariadb-10.0<br/><br/>2018-08-31<br/><br/>Debian<br/><br/>[4]DLA-1490-1<br/><br/>LTS<br/><br/>php5<br/><br/>2018-09-01<br/><br/>Debian<br/><br/>[5]DSA-4283-1<br/><br/>stable<br/><br/>ruby-json-jwt<br/><br/>2018-08-31<br/><br/>Debian<br/><br/>[6]DLA-1488-1<br/><br/>LTS<br/><br/>spice<br/><br/>2018-08-31<br/><br/>Debian<br/><br/>[7]DLA-1486-1<br/><br/>LTS<br/><br/>spice<br/><br/>2018-09-01<br/><br/>Debian<br/><br/>[8]DLA-1489-1<br/><br/>LTS<br/><br/>spice-gtk<br/><br/>2018-09-01<br/><br/>Debian<br/><br/>[9]DLA-1491-1<br/><br/>LTS<br/><br/>tomcat8<br/><br/>2018-09-02<br/><br/>Debian<br/><br/>[10]DSA-4282-1<br/><br/>stable<br/><br/>trafficserver<br/><br/>2018-08-31<br/><br/>Fedora<br/><br/>[11]FEDORA-2018-33fef25ed1<br/><br/>F28<br/><br/>ghc-hakyll<br/><br/>2018-08-31<br/><br/>Fedora<br/><br/>[12]FEDORA-2018-33fef25ed1<br/><br/>F28<br/><br/>ghc-hs-bibutils<br/><br/>2018-08-31<br/><br/>Fedora<br/><br/>[13]FEDORA-2018-07083800ac<br/><br/>F28<br/><br/>ghostscript<br/><br/>2018-09-02<br/><br/>Fedora<br/><br/>[14]FEDORA-2018-77e610115a<br/><br/>F28<br/><br/>mariadb<br/><br/>2018-08-31<br/><br/>Fedora<br/><br/>[15]FEDORA-2018-33fef25ed1<br/><br/>F28<br/><br/>pandoc-citeproc<br/><br/>2018-08-31<br/><br/>Fedora<br/><br/>[16]FEDORA-2018-f2b24ce26e<br/><br/>F28<br/><br/>phpMyAdmin<br/><br/>2018-08-31<br/><br/>Fedora<br/><br/>[17]FEDORA-2018-915602df63<br/><br/>F27<br/><br/>xen<br/><br/>2018-08-31<br/><br/>Mageia<br/><br/>[18]MGASA-2018-0366<br/><br/>6<br/><br/>java-1.8.0-openjdk<br/><br/>2018-09-02<br/><br/>Mageia<br/><br/>[19]MGASA-2018-0361<br/><br/>6<br/><br/>libarchive<br/><br/>2018-08-31<br/><br/>Mageia<br/><br/>[20]MGASA-2018-0367<br/><br/>6<br/><br/>libgd<br/><br/>2018-09-02<br/><br/>Mageia<br/><br/>[21]MGASA-2018-0356<br/><br/>6<br/><br/>libraw<br/><br/>2018-08-31<br/><br/>Mageia<br/><br/>[22]MGASA-2018-0364<br/><br/>6<br/><br/>libxcursor<br/><br/>2018-08-31<br/><br/>Mageia<br/><br/>[23]MGASA-2018-0359<br/><br/>5<br/><br/>mariadb<br/><br/>2018-08-31<br/><br/>Mageia<br/><br/>[24]MGASA-2018-0355<br/><br/>5, 6<br/><br/>mercurial<br/><br/>2018-08-31<br/><br/>Mageia<br/><br/>[25]MGASA-2018-0363<br/><br/>6<br/><br/>openssh<br/><br/>2018-08-31<br/><br/>Mageia<br/><br/>[26]MGASA-2018-0365<br/><br/>6<br/><br/>openssl<br/><br/>2018-09-02<br/><br/>Mageia<br/><br/>[27]MGASA-2018-0358<br/><br/>6<br/><br/>poppler<br/><br/>2018-08-31<br/><br/>Mageia<br/><br/>[28]MGASA-2018-0362<br/><br/>6<br/><br/>quazip<br/><br/>2018-08-31<br/><br/>Mageia<br/><br/>[29]MGASA-2018-0357<br/><br/>6<br/><br/>squirrelmail<br/><br/>2018-08-31<br/><br/>Mageia<br/><br/>[30]MGASA-2018-0360<br/><br/>6<br/><br/>virtualbox<br/><br/>2018-08-31<br/><br/>openSUSE<br/><br/>[31]openSUSE-SU-2018:2590-1<br/><br/>42.3<br/><br/>cobbler<br/><br/>2018-09-03<br/><br/>openSUSE<br/><br/>[32]openSUSE-SU-2018:2592-1<br/><br/>15.0<br/><br/>libressl<br/><br/>2018-09-03<br/><br/>openSUSE<br/><br/>[33]openSUSE-SU-2018:2587-1<br/><br/>42.3<br/><br/>wireshark<br/><br/>2018-09-02<br/><br/>openSUSE<br/><br/>[34]openSUSE-SU-2018:2591-1<br/><br/>15.0 42.3<br/><br/>zutils<br/><br/>2018-09-03<br/><br/>SUSE<br/><br/>[35]SUSE-SU-2018:2576-1<br/><br/>OS7<br/><br/>OpenStack<br/><br/>2018-08-31<br/><br/>SUSE<br/><br/>[36]SUSE-SU-2018:2578-1<br/><br/>OS7<br/><br/>couchdb<br/><br/>2018-08-31<br/><br/>SUSE<br/><br/>[37]SUSE-SU-2018:2574-1<br/><br/>SLE11<br/><br/>java-1_7_0-ibm<br/><br/>2018-08-31<br/><br/>SUSE<br/><br/>[38]SUSE-SU-2018:2583-1<br/><br/>SLE11<br/><br/>java-1_7_1-ibm<br/><br/>2018-08-31<br/><br/>SUSE<br/><br/>[39]SUSE-SU-2018:2584-1<br/><br/>SLE12<br/><br/>spice<br/><br/>2018-08-31<br/><br/><br/><br/>[1] https://lwn.net/Articles/764007/<br/><br/>[2] https://lwn.net/Articles/764008/<br/><br/>[3] https://lwn.net/Articles/764009/<br/><br/>[4] https://lwn.net/Articles/764010/<br/><br/>[5] https://lwn.net/Articles/764011/<br/><br/>[6] https://lwn.net/Articles/764013/<br/><br/>[7] https://lwn.net/Articles/764012/<br/><br/>[8] https://lwn.net/Articles/764014/<br/><br/>[9] https://lwn.net/Articles/764015/<br/><br/>[10] https://lwn.net/Articles/764016/<br/><br/>[11] https://lwn.net/Articles/764017/<br/><br/>[12] https://lwn.net/Articles/764018/<br/><br/>[13] https://lwn.net/Articles/764019/<br/><br/>[14] https://lwn.net/Articles/764020/<br/><br/>[15] https://lwn.net/Articles/764021/<br/><br/>[16] https://lwn.net/Articles/764022/<br/><br/>[17] https://lwn.net/Articles/764023/<br/><br/>[18] https://lwn.net/Articles/764024/<br/><br/>[19] https://lwn.net/Articles/764025/<br/><br/>[20] https://lwn.net/Articles/764026/<br/><br/>[21] https://lwn.net/Articles/764027/<br/><br/>[22] https://lwn.net/Articles/764028/<br/><br/>[23] https://lwn.net/Articles/764029/<br/><br/>[24] https://lwn.net/Articles/764030/<br/><br/>[25] https://lwn.net/Articles/764031/<br/><br/>[26] https://lwn.net/Articles/764032/<br/><br/>[27] https://lwn.net/Articles/764033/<br/><br/>[28] https://lwn.net/Articles/764034/<br/><br/>[29] https://lwn.net/Articles/764035/<br/><br/>[30] https://lwn.net/Articles/764036/<br/><br/>[31] https://lwn.net/Articles/764037/<br/><br/>[32] https://lwn.net/Articles/764038/<br/><br/>[33] https://lwn.net/Articles/764039/<br/><br/>[34] https://lwn.net/Articles/764040/<br/><br/>[35] https://lwn.net/Articles/764044/<br/><br/>[36] https://lwn.net/Articles/764041/<br/><br/>[37] https://lwn.net/Articles/764042/<br/><br/>[38] https://lwn.net/Articles/764043/<br/><br/>[39] https://lwn.net/Articles/764045/
	</div>
<hr/>
</div>
</body>
